src/commands/install.yml

```diff
@@ -13,48 +13,8 @@ parameters:
 steps:
   - run:
```

Comment:
The correct way to reference the `install-kops` and `install-kubectl` commands would be like this:

```
steps:
  - install-kops
  - install-kubectl
```

Reply:
Thanks @lokst. I've updated.